app/src/main/java/fr/free/nrw/commons/upload/Title.kt

```diff
@@ -14,6 +14,13 @@ class Title {
     }
 
     fun setTitleText(titleText: String?) {
+
+        if(titleText != null){
+            this.titleText = titleText.trim()
```

Comment:
Have you tried this ?

> this.titleText=titleText?.trim():null

Reply:
The method I ised seemed quite obvious and so I went with it.
Should I change the code and use the method that you mentioned?

Reply:
Yes, that would be better IMO, as the class in kotlin, it would be better if we use the benefits of the language